### PR TITLE
[ALLUXIO-2030] Add a new REST endpoint for getting mount points

### PR DESCRIPTION
**Pull request by uronce-cc**
https://alluxio.atlassian.net/browse/ALLUXIO-2030

Information about Alluxio mount points includes ufs url, ufs storage usage, ufs type, and mount point configurations.

### CODE REVIEW

**Comment by uronce-cc** [comment]
jenkins, test this please

File: core/common/src/main/java/alluxio/wire/MountPointInfo.java [hunk 1]
```diff
@@ -0,0 +1,186 @@
...
+ * This software is distributed on an "AS IS" basis, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND,
+ * either express or implied, as more fully set forth in the License.
+ *
+ * See the NOTICE file distributed with this work for information regarding copyright ownership.
+ */
+
+package alluxio.wire;
+
+import alluxio.Configuration;
+import alluxio.underfs.UnderFileSystem;
```
Comment (jsimsa) [inline core/common/src/main/java/alluxio/wire/MountPointInfo.java]: We want the common module to depend on underfs.

The wire type should simply be a set of fields with a getter and setter for each. Any non-trivial logic should be done in the caller.

Comment (jsimsa) [inline core/common/src/main/java/alluxio/wire/MountPointInfo.java]: Nevermind this comment, I checked the this class is in fact in the common package.

File: core/common/src/main/java/alluxio/wire/MountPointInfo.java [hunk 2]
```diff
@@ -0,0 +1,186 @@
+/*
```
Comment (jsimsa) [inline core/common/src/main/java/alluxio/wire/MountPointInfo.java]: Add a unit test for this class.

**Comment by jsimsa** [comment]
LGTM, but please add a unit test for `MountPointInfo`.

File: core/common/src/main/java/alluxio/wire/MountPointInfo.java (hunk 2 above)
Comment (uronce-cc) [inline core/common/src/main/java/alluxio/wire/MountPointInfo.java]: done

**Comment by jsimsa** [comment]
LGTM ... the test failure seems unrelated (@aaudiber do we know what is causing this?)

**Comment by jsimsa** [comment]
Jenkins, test this please

**Comment by uronce-cc** [comment]
jenkins, test this please

**Comment by jsimsa** [comment]
LGTM

**Pull request merged**